shenyu-common/src/main/java/org/apache/shenyu/common/utils/HmacUtils.java

```diff
@@ -59,8 +59,7 @@ public static String hmacSha512Hex(final String key, final String valueToDigest)
     }
 
     private static String getHmacHex(final HmacAlgorithms algorithm, final String key, final String valueToDigest) {
-        return new org.apache.commons.codec.digest.HmacUtils(algorithm, key)
-                .hmacHex(valueToDigest);
+        return new org.apache.commons.codec.digest.HmacUtils(algorithm, key).hmacHex(valueToDigest);
```

Comment:
why used org.apache.commons.codec.digest.HmacUtils ？

not import？

Reply:
Because the `HmacUtils` has the same name as the current class.

Reply:
maybe need renamed！

Reply:
I will change its name, please help me to CR later.